net.audiorelay.AudioRelay.yml

```diff
@@ -0,0 +1,38 @@
+app-id: net.audiorelay.AudioRelay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: audiorelay
+
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --persist=.skiko
+  - --persist=.java
+
+modules:
+  - name: audiorelay
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/audiorelay
+      - tar -xf audiorelay.tar.gz -C /app/audiorelay
```

Comment:
I don't see a JDK installed.

Reply:
The app already bundles the Java in its archive, I think it's supposed to be simply unzipped as intended, though please correct me if I'm wrong.